core/src/main/java/org/apache/brooklyn/core/sensor/ssh/SshCommandSensor.java

```diff
@@ -126,7 +118,7 @@ public String get() {
                 // Note that entity may be null during rebind (e.g. if this SshFeed is orphaned, with no associated entity):
                 // See https://issues.apache.org/jira/browse/BROOKLYN-568.
                 // We therefore guard against null in makeCommandExecutingInDirectory.
-                return makeCommandExecutingInDirectory(command, executionDir, entity);
+                return makeCommandExecutingInDirectory(params.get(SENSOR_COMMAND), params.get(SENSOR_EXECUTION_DIR), entity);
```

Comment:
I think we should do the `checkNotNull(params.get(SENSOR_COMMAND), ...)` in this `Supplier` method, and then pass it into the `makeCommandExecutingInDirectory`.

If this were null, then things can go very strange. For example, we might try to execute `null` when we construct the command via:
`finalCommand = "mkdir -p '"+execDir+"' && cd '"+execDir+"' && "+finalCommand`